### ISSUE TITLE
Status not shown in main dashboard

### ISSUE THREAD

**Issue by ariasAleia** [TwiN/gatus#656]
### Describe the bug

Main dashboard does not show the endpoints response status. I am currently using gatus in kubernetes and after port forwarding the service the main dashboard is empty. 

If I click on the back arrow I can see the endpoint but no status is shown. However, if I enter to the details I can see the status responses. 

### What do you see?

Main dashboard: 
![image](https://github.com/TwiN/gatus/assets/102924128/644c8785-d42b-4683-b540-2e8027288fb9)
After clicking to the back arrow:
![image](https://github.com/TwiN/gatus/assets/102924128/15a0597d-cfeb-41bb-9e5c-b6f1451f99b9)
Details page:
![image](https://github.com/TwiN/gatus/assets/102924128/16a088c5-4fe7-4de2-bef7-d8be7d5396c7)

### What do you expect to see?

I expect to see the status response in the main dashboard

### List the steps that must be taken to reproduce this issue

1. Port forward the gatus service
2. Navigate to localhost:8080
3. Main dashboard does not show the status responses of the endpoints

### Version

3.4.0

**Comment by TwiN** [comment]
Just out of curiosity, was it just because you didn't wait for long enough? 😂 

Gatus does cache the status for 10s, so if you visit the page as soon as the application starts, it will cache the empty results for 10 seconds: https://github.com/TwiN/gatus/blob/911902353f8bd65d374e88a801e531ae73d9b34f/api/endpoint_status.go#L44

**Comment by ariasAleia** [comment]
Thanks for your curiosity! hehe

It was not because of the waiting time. Gatus was previously working well. Then I started testing with some docker containers (from another application, basic html). One of them was using port 8080. I stopped the container, kept running various tests and after a couple of hours I wanted to test gatus again. When I visited 127.0.0.1:8080 to access gatus it presented the problem I described. Then I tried to visit localhost:8080 and I saw the application that had run in the container previously. That was strange so I removed the container and then I could access gatus from localhost:8080 but still visiting 127.0.0.1:8080 presented the same problem. 

Since I had found a workaround using localhost instead of 127.0.0.1 (after some research they usually show the same thing so I don't quite understand what happened), I kept working and the next day everything was back to normal. localhost and 127.0.0.1 on port 8080 showed the same thing.
